app/src/main/java/ch/epfl/culturequest/social/Profile.java

```diff
@@ -57,12 +97,69 @@ public String getPhoneNumber() {
         return phoneNumber;
     }
 
-    public Uri getProfilePicture() {
+    public String getProfilePicture() {
         return profilePicture;
     }
 
-    public void updateProfilePicture(Uri profilePicture) {
+    public void setUid(String uid) {
+        this.uid = uid;
+
+    }
+
+    public void setName(String name) {
+        this.name = name;
+
+    }
```

Comment:
do we need this?? a user wont change their name and everything is registered with google so i don't think there's need for this

Reply:
The thing is that Firebase need all these setters to be able to get the data from the server